packages/webstudio-sdk/.storybook/main.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
@okonet should we have the same config? how would you do it, move into a separate package?

Reply:
Depends: if we want one storybook for all packages, then we should create a separate package that would include everything. Or we do local storybooks in each package. I think both is fine right now and we can decide later.

Reply:
I'm gravitating towards separate storybooks rn since configs can be different and it will be hard to merge them IMO.